api/environments/conftest.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from environments.identities.models import Identity
+from environments.identities.traits.models import Trait
+from environments.models import Environment
+from features.feature_types import MULTIVARIATE
+from features.models import Feature
+from features.multivariate.models import MultivariateFeatureOption
+from features.value_types import INTEGER, STRING
+from organisations.models import Organisation
+from projects.models import Project
+
+
+@pytest.fixture()
+def organisation(db):
```

Comment:
Have you confirmed we don't already have any similar fixtures elsewhere for this purpose?

Reply:
Yeah, we have. I was planning to create a conftest at the root to tackle this, sounds good?

Reply:
YEp, sounds good.